payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardNumberController.kt

```diff
@@ -214,10 +221,18 @@ internal class DefaultCardNumberController(
             }
 
             val items = brands.map { brand ->
+                val enabled = cardBrandFilter.isAccepted(brand)
                 TextFieldIcon.Dropdown.Item(
                     id = brand.code,
-                    label = brand.displayName.resolvableString,
-                    icon = brand.icon
+                    label = if (enabled) {
+                        brand.displayName.resolvableString
+                    } else {
+                        brand.displayName.resolvableString +
```

Comment:
This should be pretty easy to parameterize, let me know if you want to pair on it!